pkg/gallery/query.go

```diff
@@ -2,11 +2,26 @@ package gallery
 
 import (
 	"context"
+	"fmt"
+	"github.com/stashapp/stash/pkg/job"
+	"path/filepath"
 	"strconv"
+	"strings"
 
 	"github.com/stashapp/stash/pkg/models"
 )
 
+// QueryOptions returns a GalleryQueryOptions populated with the provided filters.
```

Comment:
Related to this method definition change to accommodate the same structure for Identify Galleries as of Identify Scenes: https://github.com/stashapp/stash/pull/4423/files?diff=split&w=1#r1443579818